exosphere-shared/src/docker-helper.ls

```diff
@@ -79,13 +117,13 @@ class DockerHelper
           done!
 
 
-  @start-container = (container, done) ~>
-    new ObservableProcess("docker start -a #{container.container-name}",
-                            stdout: false,
-                            stderr: false)
-      ..on 'ended', (exit-code, killed) ->
-        | exit-code > 0 and not killed  =>  return done "Dependency #{container.container-name} failed to start, shutting down"
-      ..wait container.online-text, done
+  # @start-container = (container, done) ~>
+  #   new ObservableProcess("docker start -a #{container.container-name}",
+  #                           stdout: false,
+  #                           stderr: false)
+  #     ..on 'ended', (exit-code, killed) ->
+  #       | exit-code > 0 and not killed  =>  return done "Dependency #{container.container-name} failed to start, shutting down"
+  #     ..wait container.online-text, done
```

Comment:
Planning on cleaning up the rest of this `docker-helper.ls` file in another PR. Just wanted to make sure this function wasn't needed elsewhere before completely getting rid of it.